README.md

```diff
@@ -1,2 +1,41 @@
 # AWSClusterManagersDemo.jl
-Demo for AWSClusterManagers.jl presented at JuliaCon2021
+## Overview
+This demo project was presented at JuliaCon2021 to showcase a minimal example of the [AWSClusterManagers.jl](https://github.com/JuliaCloud/AWSClusterManagers.jl) package on AWS Batch. The job submitted to Batch calculates the approximate value of PI. The explanation for this can be found in this [article](https://jccraig.medium.com/calculate-pi-with-a-dartboard-bdb433f1c999).
+
+The job will:
+1. Launch four worker nodes
+2. Have each worker run a small process
+3. Retrieve and combine the results
+4. Display the results to CloudWatch logs
+## Setup
+**Note: When running this package you will be responsible for all AWS charges in your account.**
+On-Demand instances at the time of writing are charged on a per-minute basis. An estimated cost for running the demo project and immediately destroying the stacks is around `$0.50 USD`.
+
+To launch this yourself:
+
+1. Store your AWS account ID in an variable
+   	`AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)`
```

Comment:
```suggestion
   	```
   	AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
   	```
```
Might be more readable if these were in code blocks